src/Turn.js

```diff
@@ -0,0 +1,40 @@
+class Turn {
+  constructor(player, round) {
+    this.player = player;
+    this.round = round;
+    this.second = 0;
+    this.counter;
+    this.currentGuess;
+  }
+  updateScore(score) {
```

Comment:
I think that `updateScore` should be something that is handled by the `Player` class. Seems off to have `Turn` updating the data on an instance of `Player`.